package.json

```diff
@@ -43,23 +43,23 @@
     "release": "yarn validate && standard-version"
   },
   "peerDependencies": {
-    "jest": "<= 26",
+    "jest": "<= 27",
```

Comment:
should this really be `<= 27` ?  It means we will need a new release of this library everytime there's a new release of jest and people will have to carefully bump the libraries in tandem. I would instead just reverse the sign to `>=` the same way the `svelte` peer dependency is specified. The Jest version can be the minimum version of Jest that's supported. I don't know what that is right now. At least 26 is supported. https://github.com/mihar-22/svelte-jester/issues/25 looks like it would require Jest 27, so I'd be okay with just making it 27 now in advance

Reply:
I guess this PR is not really introducing true `>=27.x` support but more a fix for our accepted peer deps version for Jest. I'd also be happy with this PR once we change title and adhere to Ben's recommendations. We can then look at a proper PR for Jest 27 support that includes async transformers (tracked in #25).

Reply:
I updated the PR. Should we go ahead and merge it?

Reply:
Looks good. BUT I must say, that with Jest 27.x I could not get svelte-jester to run the test from my current project. I think it had to do with ESM and the way svelte-jester is written currently.

Reply:
On my machine (node 16.6.0) the tests fail with 
```
❯❯❯❯ npm run test                                                       ~/D/w/s/f/svelte-jester-27 enhancement/support-v27-of-jest

> svelte-jester@1.7.0 test
> jest src

TypeError: String.prototype.startsWith called on null or undefined
    at startsWith (<anonymous>)
    at node:internal/errors:811:19
    at Array.filter (<anonymous>)
    at node:internal/errors:809:16
    at prepareStackTrace (node:internal/errors:96:12)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1138:38)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
```

Reply:
Seems to be related to node 16.6.0 https://github.com/facebook/jest/issues/11708

Reply:
The tests run successfully with node 16.5.0 👍

Reply:
Looks like the fix for Node 16.6.0 will be released tomorrow, so we shouldn't need to block on it